app/templates/course-rounds.hbs

```diff
@@ -0,0 +1,92 @@
+<h3 class="text-center">
+  Rounds
+</h3>
+
+{{#if arrangedContent}}
+  {{#if filteredContent}}
+    <table id="rounds" class="table table-striped">
+      <thead>
+        <tr>
+          <th {{action "sortBy" "createdAt" on="click"}}>
```

Comment:
FYI, I think that `on` defaults to `click`, so you could probably drop all these `on=click`s.